presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergLogicalPlanner.java

```diff
@@ -50,6 +50,7 @@
 import static com.facebook.presto.common.predicate.ValueSet.ofRanges;
 import static com.facebook.presto.common.type.BigintType.BIGINT;
 import static com.facebook.presto.common.type.DoubleType.DOUBLE;
+import static com.facebook.presto.iceberg.IcebergAbstractMetadata.isEntireColumn;
```

Comment:
Can you please add a test testFilterPushDown similar to testParquetDereferencePushDown?